packages/esm-patient-chart-app/src/actions-buttons/start-visit.component.tsx

```diff
@@ -26,7 +26,7 @@ const StartVisitOverflowMenuItem: React.FC<StartVisitOverflowMenuItemProps> = ({
           }}
         >
           <span className="cds--overflow-menu-options__option-content">
-            {!startVisitLabel ? <>{t('startVisit', 'Start visit')}</> : startVisitLabel}
+            {startVisitLabel ? <>{t('startVisit', 'Start visit')}</> : startVisitLabel}
```

Comment:
Doesn't this do things incorrectly, i.e., if a user set the `startVisitLabel` via config, they get the translated "Start visit" screen, whereas if they did not, they get the `startVisitLabel` (which will be some kind of falsy value).

Reply:
Yes,.. I will revert that